src/types.ts

```diff
@@ -298,7 +298,7 @@ export interface RenderProps {
 
 export interface RootProps {
   direction?: 'ltr' | 'rtl';
-  selectionsAPI: stardust.ObjectSelections;
+  selectionsAPI: ExtendedSelectionAPI;
```

Comment:
I am not sure when the stardust.ObjectSelections becomes the ExtendedSelectionAPI. Looking into it.

Reply:
I explained it at above on the right file.